### ISSUE TITLE
Support for api.prompts

### ISSUE THREAD

**Issue by Aupajo** [kilterset/auth0-actions-testing#25]
```js
exports.onExecutePostLogin = async (event, api) => {
  const FORM_ID = 'ap_dULmZdtQdaq3kyqEmUfFyW';

  api.prompt.render(FORM_ID);
}
```

**Comment by Aupajo** [comment]
Fixed by #33.

**Comment by nathanKramer** [comment]
https://github.com/kilterset/auth0-actions-testing/pull/37
